app/jobs/assign_doi_job.rb

```diff
@@ -13,6 +13,7 @@ def run
     doi_request = DoiRequest.find(doi_request_id)
     asset = Collection.find(doi_request.asset_id)
     minted_doi = Ezid::Identifier.mint(
+      target: "http://data.lib.vt.edu/collections/" + asset.id,
```

Comment:
It's better, IMHO, to use a link relative to the application root rather than an absolute link like the above.